js/geojson.js

```diff
@@ -369,13 +369,15 @@ function getMap(){
 
     function createAjaxCall(neighborhood) {
         if (neighborhood === "SULLIVAN'S GULCH") {
-            // the correct way to escape a SQL apostrophe or single quote 
+            // the correct way to escape a SQL apostrophe or single quote
             // is with two single quotes
             neighborhood = "SULLIVAN''S GULCH";
         }
-        var url = "https://tcasiano.carto.com/api/v2/sql?format=GeoJSON&q=";
+
+        // TODO: Replace the backend once a free replacement for CartoDB is found
```

Comment:
My code editor auto-formatted this file, hence all the whitespace changes. The only significant change in this file is that I removed the Carto account link for the off chance that I could get charged for its use now that the free tier has ended.